sample_2011/Variable.csv

```diff
@@ -0,0 +1,567 @@
+Id,Variable_Mnemonic,Variable_Title,Variable_Title_Welsh,Variable_Description,Variable_Description_Welsh,Variable_Mnemonic_2011,Comparability_Comments,Comparability_Comments_Welsh,Uk_Comparison_Comments,Uk_Comparison_Comments_Welsh,Signed_Off_Flag,Security_Mnemonic,Variable_Type_Code,Topic_Mnemonic,Number_Of_Classifications,Statistical_Unit,Geographic_Abbreviation,Geographic_Abbreviation_Welsh,Geographic_Theme,Geographic_Theme_Welsh,Geographic_Coverage,Geographic_Coverage_Welsh,Version,Quality_Statement_Text,Quality_Summary_URL
```

Comment:
There doesn't seem to be a field for a Welsh translation of the `Quality_Statement_Text` field. Obviously not something that can be fixed in this PR though.